app.js

```diff
@@ -7,6 +7,13 @@ const govukTemplate = require('hof-govuk-template');
 const hoganExpressStrict = require('hogan-express-strict');
 const expressPartialTemplates = require('express-partial-templates');
 const _ = require('lodash');
+const apiCall = require('./lib/api-call');
+const config = require('./config');
+const baseUrl = config.apiBaseUrl;
+const surveyID = config.surveyID;
+const responses = config.endPointResponses;
+const apiToken = config.apiToken;
+const apiTokenSecret = config.apiTokenSecret;
 
```

Comment:
Good idea to move them. I wouldn't bother setting consts here though. Unless you are using them multiple times.

Personally, I'd just do:
```js
apiCall.getResponses(
  config.baseUrl,
  config.surveyID,
  config.responses,
  config.apiToken,
  config.apiTokenSecret);
```